webapp/src/components/ImportOSCAR.vue

```diff
@@ -192,12 +193,34 @@
       const operatingStatusOptions = ref(null);
 
       onBeforeMount( async() => {
-          await fetch("/code_lists/territory.json").then( (response) => response.json()).then( (data) => territoryOptions.value = data);
-          await fetch("/code_lists/facilityType.json").then( (response) => response.json()).then( (data) => facilityTypeOptions.value = data);
-          await fetch("/code_lists/operatingStatus.json").then( (response) => response.json()).then( (data) => operatingStatusOptions.value = data);
-          await fetch("/code_lists/WMORegion.json").then( (response) => response.json()).then( (data) => WMORegionOptions.value = data);
+          territoryOptions.value = await loadCodeList('territory');
+          facilityTypeOptions.value = await loadCodeList('facilityType');
+          operatingStatusOptions.value = await loadCodeList('operatingStatus');
+          WMORegionOptions.value = await loadCodeList('WMORegion');
       });
 
+      const loadCodeList = async (codeList) => {
+        try{
+          const clist = await fetch("/code_lists/" + codeList + ".json");
```

Comment:
This will fail when the proxy redirects, you need to use VITE_BASE_URL
```
var jsonURL = `${import.meta.env.VITE_BASE_URL}/code_lists/WMORegion.json`;
await fetch(jsonURL).then( (response) => response.json()).then( (data) => WMORegionOptions.value = data);
```